client/src/pages/MyProfile/MyProfile.tsx

```diff
@@ -29,7 +29,7 @@ const initSitter: Sitter = {
   profile: {
     firstName: '',
     lastName: '',
-    username: '',
+    // username: '',
```

Comment:
Can this be removed?

Reply:
addressed in another PR

Reply:
Could you elaborate on what PR this is being addressed in? If that PR removes the username prop and we are just adding a commented username prop then this is not supposed to be here. Let me know if this makes sense 😄